DelvUI/Interface/PartyCooldowns/PartyCooldownsConfig.cs

```diff
@@ -368,35 +367,57 @@ private void DrawFilter(string name, JobRoles role)
             [7520] = NewData(7520, JobIDs.RDM, 58, 120, 20, 30, 3), // embolden
             [25857] = NewData(25857, JobIDs.RDM, 86, 120, 10, 70, 2), // magick barrier
             [25801] = NewData(25801, JobIDs.SMN, 66, 120, 30, 30, 3), // searing light
+
+
+            // SINGLE-TARGET EFFECTS (disabled by default)
+
+            [30] = NewData(30, JobIDs.PLD, 50, 420, 10, 100, 4, false), // hallowed ground
+            [43] = NewData(43, JobIDs.WAR, 42, 240, 10, 100, 4, false), // holmgang
+            [3638] = NewData(3638, JobIDs.DRK, 50, 300, 10, 100, 4, false), // living dead
+            [16152] = NewData(16152, JobIDs.GNB, 50, 360, 10, 100, 4, false), // superbolide
+            [7571] = NewData(7571, JobRoles.Healer, 48, 120, 0, 80, 4, false), // rescue
+            [7561] = NewData(7561, new List<JobRoles>() { JobRoles.Healer, JobRoles.DPSCaster }, 18, 60, 0, 80, 4, false), // swiftcast
```

Comment:
swiftcast actually has a duration of 10s

Reply:
I'm not tracking the actual buff and Swiftcast is almost always used instantly. I don't really think there's much value in tracking the actual duration for this one. I could just set it to 10 duration but it wont be have accurately unless I add a layer to track the actual buff duration.

Reply:
Alright